server/src/main/java/io/druid/client/cache/MapCache.java

```diff
@@ -78,24 +96,13 @@ public CacheStats getStats()
   @Override
   public byte[] get(NamedKey key)
   {
-    final byte[] retVal;
-    synchronized (clearLock) {
-      retVal = baseMap.get(computeKey(getNamespaceId(key.namespace), key.key));
-    }
-    if (retVal == null) {
-      missCount.incrementAndGet();
-    } else {
-      hitCount.incrementAndGet();
-    }
-    return retVal;
+    return baseCache.getIfPresent(computeKey(namespaceId.getUnchecked(key.namespace), key.key));
```

Comment:
`getIfPresent` can return `null` it is a good habit to use the annotation `@Nullable`.